invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/config/templates.js

```diff
@@ -0,0 +1,7 @@
+import { ResultsItemTemplate } from "@templates/search/ResultsItemTemplate";
+import { ResultsGridItemTemplate } from "@templates/search/ResultsGridItemTemplate";
```

Comment:
I guess something has to go in the Webpack config for this to be resolved right?

Reply:
An alias in invenio-assets, latest PR :)